FinnUI/Sources/MyVehicles/MyVehiclesListView/MyVehiclesListView.swift

```diff
@@ -0,0 +1,118 @@
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+struct MyVehiclesListView: View {
+
+    // MARK: - Private
+
+    @State var navigateToPreviouslyOwnedVehicles = false
+    private let imageUrl = "https://images.finncdn.no/dynamic/default/"
+
+    // MARK: - Public
+
+    var viewModel: MyVehiclesListViewModel
+    var onTapAddNewVehicle: (() -> Void) = {}
+
+    var body: some View {
+        NavigationView {
+            List {
+                addNewVehicle
+                ForEach(viewModel.vehicles, id: \.self) { viewModel in
```

Comment:
the vehicles are identifiable, so you can write 

```
ForEach(viewModel.vehicles) { vehicle in
```

I rename the internal viewModel for vehicle so it doesn't get confused by the top view's view model